demos/sample_azure_iot/sample_azure_iot.c

```diff
@@ -512,28 +512,28 @@ static void prvAzureDemoTask( void * pvParameters )
                                                    NULL, ucMQTTMessageBuffer, sizeof( ucMQTTMessageBuffer ),
                                                    ullGetUnixTime,
                                                    &xTransport );
-        configASSERT( xResult == eAzureIoTProvisioningSuccess );
+        configASSERT( xResult == eAzureIoTSuccess );
 
         #ifdef democonfigDEVICE_SYMMETRIC_KEY
             xResult = AzureIoTProvisioningClient_SetSymmetricKey( &xAzureIoTProvisioningClient,
                                                                   ( const uint8_t * ) democonfigDEVICE_SYMMETRIC_KEY,
                                                                   sizeof( democonfigDEVICE_SYMMETRIC_KEY ) - 1,
                                                                   Crypto_HMAC );
-            configASSERT( xResult == eAzureIoTProvisioningSuccess );
+            configASSERT( xResult == eAzureIoTSuccess );
         #endif // democonfigDEVICE_SYMMETRIC_KEY
```

Comment:
Nit - prexisting, but this is only use of C++ style comment in file.  We should keep completely consistent.

Reply:
yup ran the uncrustify on it and it picked it up